### ISSUE TITLE
Spotify suddenly stops playing while still having songs in queue

### ISSUE THREAD

**Issue by MrBeardedGuy** [SpotX-CLI/SpotX-Mac#41]
### 🧐 Have you fully read the README and viewed similar issues?

- [X] I have read the README
- [X] I have checked older issues, open and closed
- [X] I did a clean install of Spotify and SpotX

### Сountry of your account

Switzerland

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: Spotify for macOS (Intel) 1.1.99.878.g1e4ccc6e 
- macOS: macOS Monterey 12.6.1
- Shell: zsh

Spotify version: 1.1.99.878
SpotX-Mac version: 1.1.99.878-2
```

### 📝 Description

1. Start Spotify
2. Listen for about 40min to 1 hour through the laptop speakers
3. Avoid any interaction with the Spotify app or the Mac touch-bar controls while the songs are playing
4. Spotify will cease to play the next song in queue after about 45min – 1h

EXPECTED
Spotify shouldn't stop the song queue but continue playing indefinitely until the user pauses it or closes the app

### 🖵 Terminal Output

N/A Terminal not involved 

### 📸 Screenshots

<img width="1778" alt="Screen Shot 2022-12-02 at 11 45 47" src="https://user-images.githubusercontent.com/36925949/205277207-7bd9aeec-54e3-4cb1-84fc-59af05dd29d8.png">

**Comment by jetfir3** [comment]
> 
> ### 🖵 Terminal Output
> N/A Terminal not involved
> 

How did you install SpotX if not with Terminal?

You are able to consistently trigger this issue with the same order of operations you've listed?
If you run the SpotX uninstaller you are no longer able to trigger the issue?

I'm unable to trigger this on my own computer... untouched computer has been playing for 3hrs now. 

I would suspect this is either a specific computer issue.. or a specific computer + spotify version issue and would try using a previous version of Spotify instead.. or perhaps an upgraded build sorts this out in the next week or so. 

These same patches are also used in the windows and linux repos and I'm not seeing the issue there as well. I'm not sure how we could fix this issue on our end so hopefully a previous or newer build is a workaround for you.

**Comment by MrBeardedGuy** [comment]
> How did you install SpotX if not with Terminal?

I did install SpotX through iTerm yes but the error wasn't related to anything in the terminal :)

> You are able to consistently trigger this issue with the same order of operations you've listed?
Yes, It happened 3 times.

> If you run the SpotX uninstaller you are no longer able to trigger the issue?
Actually no. The issue stopped. 
After I reinstalled both Spotify and SpotX from scratch I wasn't able to reproduce the problem. It might have been some weird specific problem related to my machine + Spotify configuration. 

Closing this for now since I had no issues for the past 48h after reinstalling everything.
